test/src/e2e_vm_tests/test_programs/should_pass/language/prelude_access/src/main.sw

```diff
@@ -0,0 +1,11 @@
+script;
+
+struct A {
+    addr: Address,
+}
+
+// Ensure shadowing is allowed.
+use std::address::Address;
```

Comment:
Do you mind adding another test to check that every symbol in the prelude is actually available without the need to import it?